EventFilter/OnlineMetaDataRawToDigi/python/OnlineMetaData_EventContent_cff.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+#
+# Name:   OnlineMetaData_EventContent_cff.py
+# Author: Remi Mommsen
+# Date:   20.02.2018
+
+OnlineMetaDataContent = cms.PSet(
+   outputCommands = cms.untracked.vstring(
+       'keep CTPPSRecord_metaData_*_*',
+       'keep DCSRecord_metaData_*_*',
+       'keep OnlineLuminosityRecord_metaData_*_*',
+       'keep recoBeamSpot_metaData_*_*'
```

Comment:
the module name is onlineMetaDataDigis (as defined in Configuration/StandardSequences/python/RawToDigi_cff.py).
Please use a matching name